content/4_Gold/LIS.mdx

```diff
@@ -251,7 +292,7 @@ int main() {
     cin >> n;
     for (int i = 0; i < n; i++) cin >> a[i].first >> a[i].second;
     sort(a, a + n, greater<pair<int, int>>());
-
+    //finding the LIS of a reversed array = finding the LDS
```

Comment:
This comment isn't necessary

Reply:
Why? I thought it was a little confusing when I first read through it

Reply:
I feel like this comment is okay? Might not be obvious to everyone :P